yaml-tests/src/test/resources/initial-version/at-least-version-tests.yamsql

```diff
@@ -0,0 +1,69 @@
+#
+# at-least-version-tests.yamsql
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2021-2025 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This test should pass when run on version 3.0.18.0
+---
+schema_template:
+    create table t1(id bigint, col1 bigint, primary key(id))
+---
+test_block:
+  preset: single_repetition_ordered
+  tests:
+    -
+      - query: select * from t1;
```

Comment:
There should be tests here and in `less-than-version-tests` that test against `!current_version`.
There is code in the build to exclude `supported-version` and we should do the same for this directory:
https://github.com/FoundationDB/fdb-record-layer/blob/d736d89bbde069216bef30946f1e74861387dd7d/build.gradle#L282